### PR TITLE
Adding ADR about new-instructors association to org

### PR DESCRIPTION
**Pull request by mikedikan**
DISCO-831

### CODE REVIEW

**Comment by mikedikan** [comment]
@bbaker6225 & @mikix

File: decisions/0002-new-instructor-org-association.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
+1. New Instructors Organization Association
+----------------------------------------------
+
+Status
+------
+
+Initial
```
Comment (mikix) [inline decisions/0002-new-instructor-org-association.rst]: Initial?

File: decisions/0002-new-instructor-org-association.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Consequences
+------------
+
+It is possible that users will create new instructors and not continue to save the course run that they
+are authoring.  In this case, these new instructors will be orphaned in the database (not attached to any
+organization) and will need to be cleaned up later to prevent data degradation.
+
+References
+----------
```
Comment (mikix) [inline decisions/0002-new-instructor-org-association.rst]: Could just drop this empty section? Or maybe point at the code?

File: decisions/0002-new-instructor-org-association.rst (hunk 1 above)
Comment (mikedikan) [reply decisions/0002-new-instructor-org-association.rst]: Yeah, I followed the other ADR as a template

**Pull request merged**